dmoj/checkers/identical_with_pe.py

```diff
@@ -0,0 +1,13 @@
+from dmoj.checkers._checker import standard
+from dmoj.result import CheckerResult
+from dmoj.utils.unicode import utf8bytes
+
+def check(process_output, judge_output, **kwargs):
+    # return AC iff the output matches exactly
```

Comment:
Implementation-wise this checker looks good to me, but I feel this should be integrated into the regular `identical` checker, and have PEs be opt-out rather than opt-in via a checker argument (see the `floats` checker for how `init.yml`-configured `precision` is passed in).